ksqldb-parser/src/test/java/io/confluent/ksql/parser/AstBuilderTest.java

```diff
@@ -1142,4 +1150,240 @@ public void shouldThrowOnNonIntegerId() {
     // Then:
     assertThat(e.getMessage(), is("ID must be an integer"));
   }
+
+  @Test
+  public void shouldGetCorrectLocationsSimpleCs() {
+    // Given:
+    final String query = "CREATE STREAM X WITH (kafka_topic='X');";
+    final SingleStatementContext stmt = givenQuery(query);
+
+    // When:
+    final CreateStream result = (CreateStream) builder.buildStatement(stmt);
+
+    // Then:
+    final NodeLocation loc = result.getLocation().get();
+    assertThat(loc.getStartLineNumber(), is(1));
+    assertThat(loc.getStartColumnNumber(), is(1));
+    assertThat(loc.getStopLine(), is(OptionalInt.of(1)));
+    assertThat(loc.getStopColumnNumber(), is(OptionalInt.of(query.length() - 1)));
+    assertThat(loc.getLength(), is(OptionalInt.of(query.length() - 1)));
+  }
+
+  @Test
+  public void shouldGetCorrectLocationsSimpleCsas() {
```

Comment:
nit: I don't think this test adds any extra value, we can probably remove it (having more tests isn't always a good thing, if we change something it'll just mean more work to refactor all the tests)